### PR TITLE
F639 Center map on marker tap and bounce circle. Fix stale green map circle.

### PR DESCRIPTION
**Pull request by rolflocher**

### CODE REVIEW

File: CranberryQueue/Map/MapController.swift [hunk 1]
```diff
@@ -125,19 +134,17 @@ class MapController: UIViewController, CLLocationManagerDelegate, GMSMapViewDele
...
-          marker.icon = GMSMarker.markerImage(with: UIColor.green)
-        } else {
-          marker.icon = GMSMarker.markerImage(with: UIColor(displayP3Red: 145/255, green: 158/255, blue: 188/255, alpha: 1))
-        }
+        guard let marker = locations[queueId]?["marker"] as? GMSMarker,
+            let circle = locations[queueId]?["circle"] as? TCCircle
+        else { return }
+
+        circle.fillColor = isGreen ? Colors.homeColorGreen : Colors.defaultGrayColor
+        let markerColor = isGreen ? UIColor.green : UIColor(displayP3Red: 145/255, green: 158/255, blue: 188/255, alpha: 1)
```
Comment (creiser2) [inline CranberryQueue/Map/MapController.swift]: When a guest leaves the queue, the circle stays green still.

**Comment by rolflocher** [comment]
Needs verification!!

**Comment by rolflocher** [comment]
Not ready

**Comment by rolflocher** [comment]
Ready

**Pull request merged**